### DISCUSSION TITLE
Ideas, Questions, Anything

### DISCUSSION THREAD

**Discussion by mechatroner** [mechatroner/RBQL#26] [General]
Hello World!

**Comment by kupdegrove** [comment]
This is an excellent addition to vscode. I've been using it a lot lately and love how easy it is to quickly analyze data.  I have a use case that my non-programmer mind can't seem to solve with what I can find on the internet. Many of my .csv files have quite a few fields. Is there a way to have the output appear vertically rather than horizontally?
